style.css

```diff
@@ -1,167 +1,367 @@
+  position: sticky;
+  padding-left: 12px;
+  padding-right: 12px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
 }
 
-.burger2 {
-  position: absolute;
-  top: 50%;
-  bottom: 50%;
-  left: 16.67%;
-  right: 16.67%;
-  width: 16px;
-  height: 2px;
-  background-color: #6070ff;
-  border-radius: 1px;
+.header-mobile-flex {
+  display: flex;
+  gap: 5.03px;
 }
 
-.burger3 {
-  position: absolute;
-  top: 70.83%;
-  bottom: 29.17%;
-  left: 16.67%;
-  right: 16.67%;
-  width: 16px;
-  height: 2px;
-  background-color: #6070ff;
-  border-radius: 1px;
+.header-mobile-flex img {
+  height: 17px;
 }
 
-/* Header */
+/* -------------------------- Header -------------------------- */
 
-header {
-  margin-left: 24px;
-  margin-right: 24px;
-  flex-grow: 0;
+.header {
+  position: sticky;
+  left: 0;
+  top: 0;
+  z-index: 999;
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+  width: 100%;
+  background-color: #fff;
 }
 
 .container-header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  padding-top: 3.625rem;
-  padding-bottom: 1.5rem;
 }
 
-header a {
-  font-family: "Poppins", sans-serif;
+.logo {
   font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
   line-height: 20px;
-  color: #6070ff;
   text-decoration: none;
+  color: #6070ff;
 }
 
-/* main */
-
-main {
+.burger-container {
+  width: 24px;
+  height: 24px;
   display: flex;
   flex-direction: column;
   justify-content: center;
-  align-items: flex-start;
-  flex-grow: 1;
+  gap: 3px;
+  cursor: pointer;
 }
 
-main div {
-  margin-left: 24px;
-  margin-right: 24px;
+.burger {
+  display: inline-block;
+  width: 18px;
+  height: 2px;
+  background-color: #6070ff;
+  border-radius: 1px;
+  cursor: pointer;
 }
 
-.container-main {
-  background-image: url(assets/background-mobile-version.svg);
-  background-position: -34px 0%;
-  background-size: 462px 587.15px;
+.main-background-image {
+  background-image: url(assets/BackgroundMobileOriginal.svg);
+  background-position: -34px 0;
+  background-size: cover;
   background-repeat: no-repeat;
 }
 
+/* -------------------------- Main -------------------------- */
+
 h1 {
-  margin-top: 12px;
-  margin-bottom: 12px;
-  font-family: "Poppins", sans-serif;
+  margin: 0;
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
+}
+
+h4 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+main {
+  background-color: #fff;
+  border-radius: 0 0 0 4rem;
+  min-height: 80vh;
+}
+
+.social-icons {
+  height: 20px;
+  width: auto;
+  color: #505f79;
+}
+
+.social-icons-container {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  gap: 12px;
+}
+
+/* -------------------------- Works -------------------------- */
+
+.grid {
+  padding-top: 15vh;
+  padding-bottom: 15vh;
```

Comment:
[OPTIONAL] Kindly, I suggest you use `px` instead of `vh` for the margin and padding size.